xdp-filter/test-xdp-filter.sh

```diff
@@ -261,6 +261,38 @@ test_print()
     check_run $XDP_FILTER unload $NS -v
 }
 
+run_python_test()
+{
+    python3 -c "import xdp_test_harness; import bcc;"
+    if [ $? -ne 0 ]; then
+        # Libraries are not installed.
+        return "$SKIPPED_TEST"
+    fi
+
+    # Tests are in python-tests directory in a build tree and in tests directory after install.
```

Comment:
I did not want to clutter the `xdp-filter` directory and having `tests` directory contain only the new tests seemed misleading. Maybe move all the tests (even xdpdump and xdp-loader) into `tests` directories, even though with the other tools it will contain only one file?

Reply:
Hmm, yeah having all test files in $TOOL/tests, and installing them into /usr/share/xdp-tools/tests/$TOOL seems fine. And then export the directory from the test harness so you don't need to have a conditional in the script itself...